### ISSUE TITLE
[Question] Dashboard assets not showing

### ISSUE THREAD

**Issue by drselump14** [bensheldon/good_job#282]
I've read and did all the step from https://github.com/bensheldon/good_job#dashboard, but my dashboard can't get the proper assets...... is there anything that I missed?

![Screen Shot 2021-06-25 at 09 42 04](https://user-images.githubusercontent.com/1251454/123352113-9a472b00-d599-11eb-8599-e9d9a75df35e.png)

**Comment by bensheldon** [comment]
That's not good. It looks like they are returning a 500 error. Can you share your logs that might show what is causing that 500 error?

Here's what I see on my site:

<img width="898" alt="Screen Shot 2021-06-24 at 10 11 09 PM" src="https://user-images.githubusercontent.com/47554/123372816-1f4e2800-d539-11eb-9c8e-c7c25b633385.png">

**Comment by drselump14** [comment]
ah maybe because this error?

```
ArgumentError - Before process_action callback :verify_authenticity_token has not been defined
```

**Comment by bensheldon** [comment]
@drselump14 I just released `v1.10.0` which has a fix for this. Please bundle update and let me know if that fixes your problems. Thanks again for reporting the issue 🙏

**Comment by drselump14** [comment]
Wow,, thanks for a quick fix ... really appreciate that
